app/controllers/categories_controller.rb

```diff
@@ -1,15 +1,35 @@
 class CategoriesController < ApplicationController
   before_action :set_category, only: [:show]
```

Comment:
Was this created with the thought that it would be needed somewhere else? It seems a bit odd that it is only used for one method.